ci/validate_wheel.sh

```diff
@@ -7,18 +7,17 @@ package_dir=$1
 wheel_dir_relative_path=$2
 package_name=$3
 
-RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
 
 cd "${package_dir}"
 
 rapids-logger "validate packages with 'pydistcheck'"
 
 pydistcheck \
     --inspect \
-    "$(echo ${wheel_dir_relative_path}/*.whl)"
```

Comment:
Previously `package_name` was unused in this script

Reply:
This change is why https://github.com/rapidsai/raft/actions/runs/13188022880/job/36815130064?pr=2575 is failing like this:

```text
Usage: pydistcheck [OPTIONS] [FILEPATHS]...
Try 'pydistcheck --help' for help.

Error: Invalid value for '[FILEPATHS]...': Path 'final_dist/raft-dask*.whl' does not exist.
```

`package_name` here is `raft-dask` (with a `-`), but the wheel filename begins with `raft_dask` (with a `_`). That's intentional behavior.

I think a better fix here would be to just remove `package_name` from the script (and the corresponding calls to it).

Reply:
yeah, stripped out `package_name`.  thanks for tracking that down, @jameslamb !

Reply:
No prob! It's my fault that `package_name` was left behind there unused in anyway 😅